### PR TITLE
ft: add a readme section for setup script

### PR DESCRIPTION
**Pull request by philipyoo**
lmk if I should expand upon anything or remove things

hopefully, the credentials and mention about path to script wasn't too confusing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,6 +248,55 @@ aws s3api head-object \
...
+aws_access_key_id = V28ROGMVX4C5R80TP5P3
+aws_secret_access_key = zwksPAYyx1uwyGK/ezCpPi8tS4H3NYTZrslYZeAx
+
+[backbeat-target]
+aws_access_key_id = RPHPFTBTF77MNIDF1JJ9
+aws_secret_access_key = rjTqIFBvNALJz=qgipmW/uU+I2BjU=o+WDcuOI22
+```
+
+Once credentials are setup, let's run the script to setup 2 buckets for
+replication. Below, you can specify any valid AWS bucket name for `source-bucket`
```
Comment (bennettbuchanan) [inline README.md]: If you say "valid" here, you might also refer to some doc that defines it.

File: README.md [hunk 2]
```diff
@@ -248,6 +248,55 @@ aws s3api head-object \
...
+```
+node bin/replication.js setup \
+--source-bucket source-bucket \
+--source-profile backbeat-source \
+--target-bucket target-bucket \
+--target-profile backbeat-target
+```
+
+If all went well, you should see a log message that has the message:
+`"replication setup successful"`.
```
Comment (bennettbuchanan) [inline README.md]: Maybe just include an example of the expected success output?

File: README.md [hunk 3]
```diff
@@ -248,6 +248,55 @@ aws s3api head-object \
 After some time, the object's "ReplicationStatus" should be "REPLICA".
 :smiley_cat:
 
+### More on Bucket Setup Script
+
+An optional bucket setup script is provided to quickly get 2 buckets configured
```
Comment (bennettbuchanan) [inline README.md]: You might also mention that it creates the buckets, if they don't exist.

Comment (bennettbuchanan) [inline README.md]: ~2~ two

File: README.md [hunk 4]
```diff
@@ -248,6 +248,55 @@ aws s3api head-object \
...
+[backbeat-source]
+aws_access_key_id = V28ROGMVX4C5R80TP5P3
+aws_secret_access_key = zwksPAYyx1uwyGK/ezCpPi8tS4H3NYTZrslYZeAx
+
+[backbeat-target]
+aws_access_key_id = RPHPFTBTF77MNIDF1JJ9
+aws_secret_access_key = rjTqIFBvNALJz=qgipmW/uU+I2BjU=o+WDcuOI22
+```
+
+Once credentials are setup, let's run the script to setup 2 buckets for
```
Comment (bennettbuchanan) [inline README.md]: ~2~ two

**Review by bennettbuchanan — DISMISSED** [review]
Thanks for this, looks good!

In the help output of the script I've noticed some options need to be marked as 'required'. For example,

```
  Options:

    --source-bucket <name>   [required] source bucket name
    --source-profile <name>  aws/credentials profile to use for source
    --target-bucket <name>   target bucket name
    --target-profile <name>  aws/credentials profile to use for target
    -h, --help               output usage information
```

Also, it's pretty minor but the help option for the script itself lists `[options]` first, then `[command]`:

```
Usage: replication [options] [command]
```

But then the usage of a particular command is reversed:

```
 Usage: setup [options]
```

**Comment by philipyoo** [comment]
@bennettbuchanan Good catch! Will push up the changes for the script 'required' options soon. Is it ok to add these changes to this pr or should I make a separate pr?

Also, for the output when viewing the help stuff for the script itself (without commands) and with a specific command looks different. The script itself has a few optional options or commands, and each command have their own separate options as well

**Comment by bennettbuchanan** [comment]
I think a separate commit in this PR would work.

**Review by jonathan-gramain — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -251,6 +251,71 @@ aws s3api head-object \
...
+not already exist) and/or configure two buckets for replication. This section
+will expand upon the details of the script and what it has to offer.
+
+As a note, this script was used above to setup replication on buckets. The
+example commands provided in this section **will assume you run them from the
+root directory** of this Backbeat project.
+
+To use this script, we require you to have setup one or two AWS profiles within
+the `~/.aws/credentials` file. You can follow the [AWS create-access-key Guide](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/)
+to create your own AWS keys. If our Vault service is available to you, we
```
Comment (LaureVergeron) [inline README.md]: I would make it explicit that this comes with the Zenko Enterprise Edition?

**Comment by LaureVergeron** [comment]
LGTM, just one suggestion :)

**Comment by philipyoo** [comment]
closing for now as a bit out-of-date and would like to re-organize readme

**Pull request closed without merging**